src/lib.rs

```diff
@@ -48,18 +48,18 @@ impl Logger {
 }
 
 struct StartTime;
-impl Key for StartTime { type Value = time::Instant; }
+impl Key for StartTime { type Value = time::Tm; }
```

Comment:
The problem with `time::Instant` is that I can't get the values from it, the *seconds* and *nanoseconds*.

It may be because I don't understand Rust 100% yet but I can't find a way to do it without `time::Tm`.